tests/e2e/e2e_test.go

```diff
@@ -41,6 +41,7 @@ const (
 	defaultReportDir            = "_report/"
 	clusterProvisioningToolKey  = "CLUSTER_PROVISIONING_TOOL"
 	clusterProvisioningToolCAPZ = "capz"
+	testMultiSLB                = "TEST_MULTI_SLB"
```

Comment:
I guess you are going to add a pipeline for multi slb, right? Then why not add LABEL_FILTER in your pipeline?

Reply:
I want to skip these cases for the tests we have currently without touching the job yaml.

Reply:
I don't have strong opinion and ok with this change.
But this doesn't sound like a best practice because there's a gap between LABEL_FILETER in job yaml and test description if they don't check this file.